### DISCUSSION TITLE
close modal issue if loop 2 modals open

### DISCUSSION THREAD

**Discussion by vadimwe** [wire-elements/modal#283] [Q&A]
There is a bug with close modal if loop 2 modals open multiple times (3+).
After close all modals backdrop still persist

**Comment by PhiloNL** [comment]
Please make sure you don't include Alpine more than once.

**Reply by vadimwe** [reply]
Alpine is loaded once in layout footer...

**Reply by PhiloNL** [reply]
I can't reproduce this issue, I would recommend creating a project from scratch and checking if you experience the same problem. If the problem doesn't exist, it has to do with your existing project setup.
